vm/src/object/ext.rs

```diff
@@ -212,6 +212,49 @@ impl<T: PyPayload> ToPyObject for PyRefExact<T> {
     }
 }
 
+pub struct PyAtomicRef<T: PyObjectPayload>(AtomicCell<NonNull<Py<T>>>);
```

Comment:
Will using `AtomicCell<NonNull>` spend same cost as `AtomicPtr`?

Reply:
[AtomicCell::<T>::is_lock_free()](https://docs.rs/crossbeam/latest/crossbeam/atomic/struct.AtomicCell.html#method.is_lock_free) should be true for the platforms that support AtomicPtr. But I am not sure about any overhead that AtomicCell does.